tests/test_plotting.py

```diff
@@ -388,3 +388,11 @@ def test_stack_plot_other(plot_df):
          .stack_plot(ax=ax, stack='scenario', cmap='viridis', title='foo')
          )
     return fig
+
+
+@pytest.mark.skipif(IS_WINDOWS, reason=WINDOWS_REASON)
```

Comment:
this needs to be deleted now. apologies, I was thrashing in CI and removed this from other functions.